docs/src/examples/Cloudy_example.md

```diff
@@ -0,0 +1,518 @@
+y_mean, y_var = Emulators.predict(
+    emulator,
+    reshape(θ_true, :, 1);
+    transform_to_real = true
+)
+
+println("Emulator ($(case)) prediction on true parameters: ")
+println(vec(y_mean))
+println("true data: ")
+println(truth_sample) # what was used as truth
+```
+
+The emulator predicts both a mean value and a covariance.
+
+### Sample the posterior distributions of the parameters
+
+The last step is to plug the emulator into an MCMC algorithm, which is then used to produce samples from the posterior distribution of the parameters. Essentially, the emulator acts as a stand-in for the original forward model (which in most cases of interest is computationally expensive to run) during the MCMC sampling process.
+
+We use the mean across all parameter ensembles from the EKI as the initial
+parameters for the MCMC. Before running the actual MCMC chain, we determine a
+good step size:
+
+```julia
+# initial values
+u0 = vec(mean(get_inputs(input_output_pairs), dims = 2))
+println("initial parameters: ", u0)
+
+# First let's run a short chain to determine a good step size
+yt_sample = truth_sample
+mcmc = MCMCWrapper(
+    RWMHSampling(),
+    truth_sample,
+    priors,
+    emulator;
+    init_params = u0
+)
+
+new_step = optimize_stepsize(
+    mcmc;
+    init_stepsize = 0.1,
+    N = 2000,
+    discard_initial = 0
+)
+```
+
+We choose a sample size of 100'000 for the actual MCMC, discarding the first
+1'000 samples as burn-in:
+
+```julia
+# Now begin the actual MCMC
+println("Begin MCMC - with step size ", new_step)
+chain = MarkovChainMonteCarlo.sample(
+    mcmc,
+    100_000;
+    stepsize = new_step,
+    discard_initial = 1_000
+)
+```
+
+After running the MCMC, we can extract the posterior samples as follows:
+
+```julia
+posterior = MarkovChainMonteCarlo.get_posterior(mcmc, chain)
+```
+
+The samples of the posterior distributions represent the ultimate output of the
+CES process. By constructing histograms of these samples and comparing them with
+the known true parameter values, we can evaluate the results' accuracy. Ideally,
+the peak of the posterior distribution should converge near the true values,
+indicating a high-quality estimation. Additionally, visualizing the prior
+distributions alongside the posteriors shows the distributional change effected by the Bayesian learning process.
+
+### Results
+The following three plots show the posterior distributions obtained by the CES
+algorithm, for the inverse problem of finding the parameters ``N_{0,
+0}``, ``k_0``, and ``\theta_0``, which define a gamma distribution of droplet
+masses in Cloudy at time ``t = 0``. The results are shown in the unbounded space
+where the CES algorithm takes place. The samples posterior distributions are
+shown as grey histograms, the parameters' prior distribution are shown in blue,
+and the true parameter values are marked as vertical green lines.
+
+![posterior_N0_rf](../assets/posterior_rf-scalar_N0.png)
```

Comment:
could we plot both the rf-scalar and a GP run here? If GPJL has issues, maybe we can use scikit-learn?

Reply:
Yes we can, I just thought that showing a corner plot and the three marginals for each emulator would be a bit of a plot overload, especially given that readers of the docs will probably be interested more in the functionality / step-by-step guidance than in the results.
In the current version of the docs, I show the corner plots for both GP and RF, and the three marginals for the GP case---let me know if you think we should add more!